contracts/Everlong.sol

```diff
@@ -348,14 +404,21 @@ contract Everlong is IEverlong {
     ///         - The current idle liquidity is above the target.
     /// @return True if the portfolio can be rebalanced, false otherwise.
     function canRebalance() public view returns (bool) {
-        uint256 balance = ERC20(_asset).balanceOf(address(this));
-        uint256 target = targetIdleLiquidity();
-        return (hasMaturedPositions() ||
-            (balance > target &&
-                balance - target >
+        return hasMaturedPositions() || canOpenPosition();
+    }
+
+    /// @notice Returns whether Everlong has sufficient idle liquidity to open
+    ///         a new position.
+    /// @return True if a new position can be opened, false otherwise.
+    function canOpenPosition() public view returns (bool) {
+        uint256 balance = IERC20(_asset).balanceOf(address(this));
+        uint256 max = maxIdleLiquidity();
+        return
+            balance > max &&
```

Comment:
Shouldn't this be `balance > targetIdleLiquidity()`? That's what it used to be. What's the reason why this changed?

Reply:
as I was rewriting this block I realized `maxIdleLiquidity` wasn't really being used anywhere else and thought this might be a good purpose for it.

In my head, target idle is the goal for rebalancing, and max idle is the upper limit before a rebalance should occur. The values between target and max offer a "safe zone" of operation. With this model, it probably makes sense to include a `minIdleLiquidity` where below that triggers a sale of unmatured positions during rebalance.

Probably worth jamming on this a bit in the coming weeks... 

Worth noting that as I was going through the start of the yearn docs, it seemed like it may make sense to handle idle at the yearn vault/strategy layer, so maybe we wait on the discussion until after that is more flushed out?